OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -224,3 +302,7 @@ def toggleTransducerTrackingApproval(self) -> None:
             raise RuntimeError("Cannot toggle tracking approval because there is no active session.")
         session.toggle_transducer_tracking_approval() # apply the approval or lack thereof
         data_parameter_node.loaded_session = session # remember to write the updated session object into the parameter node
+
+    def runTransducerTracking(self) -> None:
```

Comment:
(I am just looking quickly here, this doesn't look like it has the right function signature given how it's called above)

Reply:
Oh you're right, I missed updating that